src/com/owncloud/android/notifications/NotificationBuilderWithProgressBar.java

```diff
@@ -55,7 +55,8 @@
         if (Build.VERSION.SDK_INT < Build.VERSION_CODES.ICE_CREAM_SANDWICH) {
             return new NotificationBuilderWithProgressBar(context); 
         } else {
-            return new NotificationCompat.Builder(context);
+            return new NotificationCompat.Builder(context).
+                    setColor(context.getResources().getColor(R.color.primary));
```

Comment:
@rperezb primary is the action bar color :) - with the move to material design the primary color defines the color of the action bar. the color you referred to isn't used anymore in the app. So this is right now legacy code looking at android app but might still be used by other third party apps who haven't moved to material with the AppCompat implementation yet.

In case we want to define this for third party apps also would mean we would have to move the primary color etc to the setup.xml file